utility/src/test/java/org/oppia/util/logging/firebase/FirebaseEventLoggerTest.kt

```diff
@@ -0,0 +1,204 @@
+package org.oppia.util.logging.firebase
```

Comment:
To me this entire test file isn't really making sense. The name suggests that they are tests for the firebaseEventLogger. But we are not actually testing the actual firebaseEventLogger, we are testing the fake. Which is already done in the FakeEventLoggerTest
So what is the goal of this test class? If you want to test the actual FIrebaseEventLogger, you would have to initialize it with a mock of the Firebase instance.

Reply:
From what I think, this test file was to check if the bundles are being successfully created using the proto objects or not. Taking the example of the FirebaseExceptionLogger, we weren't testing it as it was just a wrapper, is there a need to do this here then ?

Reply:
Discussion done offline.